samples/embedded/README.md

```diff
@@ -0,0 +1,66 @@
+# Embedded Example
+
+This example shows how to
+
+1. Build a PyTorch functional model for training
```

Comment:
Let's pick a more descriptive name than "embedded" for this sample.

Brainstorming other words to include:
* Training
* PyTorch
* Native
* Deployable

Reply:
Updated to "Native Training"